src/app/pages/dashboard/widgets/system/common/product-image/product-image.component.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Component, ChangeDetectionStrategy, input,
+  computed,
+  HostBinding,
+} from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'ix-product-image',
+  templateUrl: './product-image.component.html',
+  styleUrls: ['./product-image.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ProductImageComponent {
+  @HostBinding('class.ix-logo') get isIxLogo(): boolean {
+    return this.productImage() === 'ix-original.svg';
+  }
+  @HostBinding('class.truenas') isEnterprise = input.required<boolean>();
+  product = input.required<string>();
+  productImage = input.required<string>();
+  hasEnclosureSupport = input.required<boolean>();
+  isHaLicensed = input.required<boolean>();
+  isUnsupportedHardware = input.required<boolean>();
+  isCertified = input.required<boolean>();
+
+  productImageSrc = computed(() => {
+    if (!this.productImage()) {
```

Comment:
I think it's better to call the signal once and assign the value to a variable const and reuse that instead of calling it over and over for readibility. Also, although not applicable here, conditional signals can sometimes be ignored in certain cases.